packages/server/src/distributed/local/subservice.ts

```diff
@@ -15,6 +15,10 @@ export class LocalSubservice implements DistributedSubservice {
 
   async listen(channel: string, callback: (message: any) => Promise<void>) {}
 
+  async unsubscribe(channel: string) {}
+
+  async publish(channel: string, message: any) {}
```

Comment:
It would be nice to have typings for the message here. Something generic like this that allows the caller to defined its own typings:

```suggestion
  async publish<Message = any>(channel: string, message: Message) {}
```

Reply:
We have typings now for the Dispatcher. I don't see much use in putting typings in the DistributedService, as this service doesn't know what it has to send. As far as it's concerned it's sending some payload.